### PR TITLE
Add name to location fixtures index

### PR DESCRIPTION
**Pull request by esoergel**
https://trello.com/c/lgmg71xz/222-locations-fixture-index-updates

### CODE REVIEW

**Comment by esoergel** [comment]
@mkangia @ctsims

**Comment by mkangia** [comment]
The build failure was not related.. retriggered to confirm if that was a temp error

File: corehq/apps/locations/fixtures.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def get_xml_nodes(self, fixture_id, restore_user, locations_queryset, data_field
         )
         location_type_attrs = ['{}_id'.format(t) for t in all_types if t is not None]
         attrs_to_index = ['@{}'.format(attr) for attr in location_type_attrs]
-        attrs_to_index.extend(['@id', '@type'])
+        attrs_to_index.extend(['@id', '@type', 'name'])
```
Comment (mkangia) [inline corehq/apps/locations/fixtures.py]: What decides if we need '@' with the name or not?

**Review by mkangia — APPROVED** [review]
Looks good. Just one minor point to clarify

File: corehq/apps/locations/fixtures.py (hunk 1 above)
Comment (ctsims) [reply corehq/apps/locations/fixtures.py]: These are acccessors into the xml element, so anything with an @ is an attribute of the root element, and things with no @ are child elements

Comment (esoergel) [reply corehq/apps/locations/fixtures.py]: Whether it's an xml attribute or a child node.  If you expand the example test file, you'll see that id, type, and the others are attrs, and name is a child node.

**Comment by esoergel** [comment]
Gonna close in favor of https://github.com/dimagi/commcare-hq/pull/17921, which builds off of this

File: corehq/apps/locations/fixtures.py (hunk 1 above)
Comment (mkangia) [reply corehq/apps/locations/fixtures.py]: Much Thanks! 👍

**Pull request closed without merging**